### ISSUE TITLE
still doesn't cover removing the plugin from the outlook desktop application

### ISSUE THREAD

**Issue by RCGms** [MicrosoftDocs/OfficeDocs-WorkplaceAnalytics#113]
I've been to the Myanalytics page, more than a week ago, and still this Insights button displays.  I can't find a com add-in to disable, and removing it via customizing the ribbon, doesn't remove it permanently, closing and re-opening outlook brings it right back up.  And yes on any given work day I have to close and re-open outlook 3 or 4 times to completely switch profiles I'm using.  (another long time complaint, where outlook use to allow you to have multiple profiles open in specific outlook window instances)

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 74c447e0-fae6-a016-c6bf-01438701ce17
* Version Independent ID: b6d82917-aee4-2808-00ce-df6c20b75766
* Content: [Insights Outlook add-in - Workplace Intelligence](https://docs.microsoft.com/en-us/workplace-analytics/myanalytics/use/add-in#to-remove-insights-from-outlook)
* Content Source: [WorkplaceAnalytics/MyAnalytics/Use/add-in.md](https://github.com/MicrosoftDocs/OfficeDocs-WorkplaceAnalytics/blob/public/WorkplaceAnalytics/MyAnalytics/Use/add-in.md)
* Product: **mya**
* GitHub Login: @paul9955
* Microsoft Alias: **madehmer**

**Comment by paul9955** [comment]
Hi RCGms,

I looked into this and found an answer that I hope solves your problem. I added it to our documentation and you can find it here: [Opt out of MyAnalytics / If you have multiple accounts](https://docs.microsoft.com/en-us/workplace-analytics/myanalytics/use/opt-out-of-mya#if-you-have-multiple-accounts). 
Please let me know if this does not work for you.
Thank you,
 -- paul9955, MyAnalytics documentation

**Comment by RCGms** [comment]
@paul9955, thank you, but no, its the same type of instruction.  accessing your o365 instance and turning off the individual notifications or all notifications.

I did finally get it resolved today, however, because an email came in with a recognized "unsubscribe" option.  I say "recognized" because it triggered a web-add-in that would allow you to click an unsubscribe button that appears as a bar/closed menu section, just a little smaller height-wise than the close/comment buttons here on GitHub.... the section appears inbetween the preview pane's message header (top visible half of message preview pane) and the bottom/body of the message in the preview pane.

to the far right of this "unsubscribe" button was a "get more add-ins" or something, and clicking that took me to the web interface for adding/removing plugins to Outlook.  While looking around, there is a "My - Add ons" section, and looking there, were several different add-ons I would never choose, and surprise .. one for Insight, with it check-marked as installed.

I removed several of these default (defaulted to me) optional plug-ins and this removed the Insite icon from my outlook 2016 desktop app.

The same Web-add-in's web page is now reachable by the "Get Add-ins" ribbon bar button (looks like a 4-pane window box with the upper right pane replaced with a + symbol (hopefully this link comes through:  https://www.dropbox.com/s/nmgmxm8fjhjpybx/Screenshot%202020-05-06%2001.14.11.png?dl=0   ) 
In times past, when I'd click that button there was no plug-in there, perhaps that's a change for Office/outlook recently.

Anyway .. gone now finally .. I wouldn't be so upset if it would allow both methods, simply removing from the Ribbon fails . and that's what sets this whole thing in motion.
